wdl/wdl-core/src/main/scala/wdl/expression/WdlStandardLibraryFunctions.scala

```diff
@@ -25,7 +29,16 @@ trait WdlStandardLibraryFunctions extends WdlFunctions[WomValue] {
 
   def stderr(params: Seq[Try[WomValue]]): Try[WomFile]
 
-  def size(params: Seq[Try[WomValue]]): Try[WomFloat]
+  def singleFileSize(file: WomValue): Try[Double]
+
+  /**
+    * Gets the size of a file.
+    *
+    * @param params First parameter must be a File or File? or coerceable to one. The second is an optional string containing the size unit (eg "MB", "GiB")
+    */
+  def size(params: Seq[Try[WomValue]]): Try[WomFloat] = {
+    wdlVersionSpecifics.sizeFunctionOverride(params, singleFileSize)
```

Comment:
This is the hole in `wdl-core` which relies on there being a `wdlVersionSpecifics` injected.